### PR TITLE
follow pep8 guidelines

### PR DESCRIPTION
**Pull request by malverick**

### CODE REVIEW

**Comment by dessant** [comment]
@malverick, the changes cause at least one python syntax error, and introduce a couple of pep8 errors which are currently ignored, mostly E128. But we won't tell it where. 😋

File: examples/demo/multistroke/gesturedatabase.py [hunk 1]
```diff
@@ -134,7 +134,6 @@ def unload_gestures(self, *l):
             if i.ids.select.state == 'down':
                 self.selected_count -= 1
                 for g in i.gesture_list:
-#                    if g in self.recognizer.db:  # not needed, for testing
```
Comment (malverick) [inline examples/demo/multistroke/gesturedatabase.py]: done

**Comment by dessant** [comment]
@malverick, I've seen your ping on irc asking about the issues I was referring to, though I haven't seen *any* of them being fixed before asking for further guidance.

I believe being a gsoc student requires at least initiating action and attempting to discover your mistakes, it can't be expected that maintainers will micromanage everything, including every single syntax error.

These changes introduce pep8 issues:
https://github.com/kivy/kivy/pull/4822/files#diff-81918be91ba63f929fab4570f1bd005cR1085
https://github.com/kivy/kivy/pull/4822/files#diff-81918be91ba63f929fab4570f1bd005cR1069
https://github.com/kivy/kivy/pull/4822/files#diff-ed66fa7317c70c7e7d9afa9867c5e0dcR63

The printed string will contain the spaces used for indentation:
https://github.com/kivy/kivy/pull/4822/files#diff-2d79b53f5ac75b34374ce991529310feR205

Syntax errors/bugs:
https://github.com/kivy/kivy/pull/4822/files#diff-a5080129bf15b963d770e884609e7c55R61
https://github.com/kivy/kivy/pull/4822/files#diff-b79c5046620071b699db11009469a22aR121

@malverick, @saqib1707, @phunsukwangdu: thanks for your contributions so far guys, we will fix the remaining style issues. Please make sure that future prs don't delete previously merged commits, introduce unrelated code, or contain multiple issues like the ones listed above. Reading the changes a couple of times, even after the pr has been submitted, can make a real difference.

**Comment by malverick** [comment]
@dessant, I am really sorry for not looking thoroughly to my commits and for not discovering these mistakes on my own. I will definitely remember this from now on and will explicitly mention when my PR will be ready to merge and will go through my commits multiple times after submitting pr.
Please accept my apologies and thank you for your guidance. I am looking forward to learn a lot from you all.

**Comment by dessant** [comment]
@malverick, it's ok, don't be discouraged. 😋 I thought it would be good to clarify our position from the beginning, since we've seen these issues in the prs of all mentioned students.

**Comment by malverick** [comment]
@dessant, yeah I appreciate that.

**Pull request closed without merging**